org.unofficialrevport.REVHubInterface.yml

```diff
@@ -0,0 +1,69 @@
+id: org.unofficialrevport.REVHubInterface
```

Comment:
is `unofficialrevport.org` associated with the upstream developers?

Reply:
Yes

Reply:
it's not obvious. It's linked nowhere and never mention the package.

Reply:
The website is still being made

Reply:
Getting the website up and linking the software would be nice, I'm marking it as blocked until that is done.

Reply:
The website isn't empty anymore, though more content will be added in the near(ish) future.